webstruct/wapiti.py

```diff
@@ -72,6 +74,37 @@ def create_wapiti_pipeline(model_filename=None,
     ])
 
 
+def merge_top_n(chains):
+    """
+    >>>
+    >>> chains = [ ['B-PER', 'O'     ],
+    ...            ['O'    , 'B-FUNC'] ]
+
+    >>> merge_top_n(chains)
+    ['B-PER', 'B-FUNC']
```

Comment:
Could you please add doctests for more trickier cases, to make it clear how it works? E.g. what happens if entities overlap, or partially overlap?

Reply:
Added test cases and description